Formula/python-tabulate.rb

```diff
@@ -17,12 +17,13 @@ class PythonTabulate < Formula
   end
 
   depends_on "python@3.10" => [:build, :test]
-  depends_on "python@3.11" => [:build, :test]
   depends_on "python@3.9" => [:build, :test]
+  depends_on "python@3.11"
```

Comment:
Will need an audit fix first:
```
python-tabulate:
  * 19: col 14: References to `python@3.10` should match the specified python dependency (`python@3.11`)
  * 20: col 14: References to `python@3.9` should match the specified python dependency (`python@3.11`)
```

We intentionally allowed skipping Python from versioned audit, but the new Python audit has made this fail again when we have this scenario of making one runtime